src/components/mapComp/GoogleMapComp.jsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from 'react';
+import { GoogleMap, Marker, withGoogleMap, withScriptjs } from 'react-google-maps';
+
+import { readAll } from '../../utils/api/read';
+import getCoordsFromAddress from '../../utils/services/getCoordsFromAddress';
+import styles from './mapComponent.module.scss';
+
+const API_KEY = import.meta.env.VITE_GOOGLE_API_KEY;
+
+function Map() {
+  const [housesCoords, setHousesCoords] = useState([]);
+
+  useEffect(() => {
+    // eslint-disable-next-line no-undef
+    const geocoder = new google.maps.Geocoder();
+    if (!housesCoords.length) getCoordsFromAddress(readAll('houses'), geocoder, setHousesCoords);
```

Comment:
If `houses` are passed as props, `readAll('houses')` could be replaced with `houses`.

Reply:
You could also check if `houses.length`. If not, it could call `readAll('houses')` but for now it might be enough to just rely on props.